workspace/src/app/views/shared/modals/CreateArtefactModal/ArtefactForms/ParameterAutoCompletion.tsx

```diff
@@ -168,13 +173,33 @@ export const ParameterAutoCompletion = ({
 
     const itemValue = (value: StringOrReifiedValue) => (typeof value === "string" ? value : value.value);
 
+    const handleSearch = React.useCallback(
+        (input: string) => {
+            setSearchQuery(input);
+            setLimit(AUTOCOMPLETION_LIMIT)
+            setShouldLoadMoreResults(true)
```

Comment:
A second request will still be made even though the first request was less than the limit.